docs/source/tutorial_fast.md

```diff
@@ -2,50 +2,65 @@
 
 Author: Stephen Roller
 
-This tutorial walks you through a few ways to massively speed up your training runs
-in ParlAI. These tricks tend to work best with generative models, but some can also be
-used with others.
+This tutorial walks you through a few ways to massively speed up your training
+runs in ParlAI. These tricks tend to work best with generative models, but some
+can also be used with others.
+
+:::{warning} Retuning-hyperparameters
+Many of these options are effectively tricks for changing the batchsize.
+Note that you will want to freshly tune your learning rate when using
+these tricks, or simply use these options in the first place.
+
+This tutorial is only for illustration purposes of how to speed up training,
+and may not get you the best _performing_ model.
```

Comment:
assuming this is just a standard disclaimer? or d you actually see worse performance?

Reply:
The LR isn't optimized for this, and many of them don't train for remotely long enough.